pandas/core/indexes/interval.py

```diff
@@ -689,6 +689,39 @@ def _find_non_overlapping_monotonic_bounds(self, key):
         return start, stop
 
     def get_loc(self, key, method=None):
+        """Get integer location for requested label.
+
+        Parameters
+        ----------
+        key : label
+        method : {None, 'pad'/'ffill', 'backfill'/'bfill', 'nearest'}, optional
```

Comment:
I wouldn't tlist the other methods, as you say they are not implemented (like you did in the categorical docstring)

Reply:
Ok